README.md

```diff
@@ -259,7 +275,7 @@ cargo test
 
 ## Minimum requirements
 
-- GCC 4.9 or higher is required (if parsing coverage artifacts generated by GCC).
+* GCC 4.9 or higher is required (if parsing coverage artifacts generated by GCC).
```

Comment:
Why this change?

Reply:
It was just to use the same character everywhere, this isn't really necessary.

Reply:
OK, WFM